Tests/Data/Parabolic/T/3D_3BHEs_array/pre/3bhes.py

```diff
@@ -0,0 +1,128 @@
+###
+# Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+# Distributed under a Modified BSD License.
+# See accompanying file LICENSE.txt or
+# http://www.opengeosys.org/project/license
+###
+
+from tespy import cmp, con, nwk, hlp
+from tespy import nwkr
+from sub_btes import btes_para as bp
+
+import numpy as np
+import pandas as pd
+# %% network
+btes = nwk.network(fluids=['water'], T_unit='K', p_unit='bar', h_unit='kJ / kg', T_range=[273.25, 373.15], p_range=[1, 20], h_range=[1, 1000])
+
+#components
+fc_in = cmp.source('from consumer inflow')
+fc_out = cmp.sink('from consumer outflow')
+
+pu = cmp.pump('pump')
+
+sp = cmp.splitter('splitter',num_out = 3)
+
+#bhe:
+bhe_name = 'BHE1'
+assert 'BHE1' in bhe_name, "BHE should be named with 'BHE1'"
+bhe1 = cmp.heat_exchanger_simple(bhe_name)
+bhe_name = 'BHE2'
+assert 'BHE2' in bhe_name, "BHE should be named with 'BHE2'"
+bhe2 = cmp.heat_exchanger_simple(bhe_name)
+bhe_name = 'BHE3'
+assert 'BHE3' in bhe_name, "BHE should be named with 'BHE3'"
+bhe3 = cmp.heat_exchanger_simple(bhe_name)
+
+mg = cmp.merge('merge',num_in = 3)
+
+cons = cmp.heat_exchanger_simple('consumer')
+
+#connections
+# inlet
+fc_pu = con.connection(fc_in, 'out1', pu, 'in1')
+
+pu_sp = con.connection(pu, 'out1', sp, 'in1')
+
+sp_bhe1 = con.connection(sp, 'out1', bhe1, 'in1')
+sp_bhe2 = con.connection(sp, 'out2', bhe2, 'in1')
+sp_bhe3 = con.connection(sp, 'out3', bhe3, 'in1')
+
+bhe1_mg = con.connection(bhe1, 'out1', mg, 'in1')
+bhe2_mg = con.connection(bhe2, 'out1', mg, 'in2')
+bhe3_mg = con.connection(bhe3, 'out1', mg, 'in3')
+
+mg_cons = con.connection(mg, 'out1', cons, 'in1')
+
+cons_fc = con.connection(cons, 'out1', fc_out, 'in1')
+
+btes.add_conns(fc_pu, pu_sp,
+sp_bhe1, sp_bhe2, sp_bhe3,
+bhe1_mg, bhe2_mg, bhe3_mg,
+mg_cons, cons_fc)
+
+# busses
+heat = con.bus('consumer heat demand')
+heat.add_comps({'c': cons, 'p': 'P'})
+btes.add_busses(heat)
+
+# flow_char
+# provide volumetric flow in m^3 / s
+x = np.array([0.00, 0.001952885971862, 0.00390577194372, 0.005858657915586,
+  0.007811543887448, 0.00976442985931, 0.011717315831173,
```

Comment:
do some python code formatting too, please. Here are 2 spaces, above there are 4 spaces. Take 4, be consistent.
Do it automatically, not by hand.

Reply:
Completed in [d3d3e3](https://github.com/ufz/ogs/pull/2702/commits/d3d3e3ab5a70ba7be9fd3c632e729f71c0e922c1)